src/theme/components/searchBar.ts

```diff
@@ -10,14 +10,34 @@ const SearchBar = defineMultiStyleConfig({
       base: "xs",
       md: "auto",
     },
-    flexFlow: {
-      base: "column nowrap",
-      md: "row nowrap",
-    },
     ".textInput": {
-      flexGrow: "1",
+      flexGrow: 1,
+      "div > input": {
+        borderRightRadius: 0,
+      },
+    },
+    flexFlow: { base: "column nowrap", md: "row" },
+    ".searchButton": {
```

Comment:
Why did you switch from the previous `button` keyword to the class name? CSS precedence rules?

Reply:
Yes styles were getting applied to the clear button

Reply:
That's annoying. I know there are other instances of class name as the key in the them object but I'd like to move away from that so we don't use class names. Leave this as is but keep that in mind.